packages/core/src/utilities.ts

```diff
@@ -63,3 +64,10 @@ export function applyStyles(
 
   return flattenedStyles;
 }
+
+export function getRealValue(value: string | number | undefined) {
+  if (isString(value)) return value;
```

Comment:
just some formatting / style:

```
if (isString(value)) {
  return value
}

if (isNumber(value)) {
  return String(value)
}
```

not sure why the eslint stuff isn't picking up here in jigsaw (perhaps its missing) but in the builder we don't allow 1 line if statements

wrt else if: since the previous if block is already returning, you can avoid the `else if`

Reply:
@peter I will keep it consistent with people that style if + if else + else